### ISSUE TITLE
Tiny fix : ofLog::padding -> ofLog::getPadding

### ISSUE THREAD

**Issue by elliotwoods** [openframeworks/openFrameworks#5017]
Currently there's a static string in the class

ideally we change this to a static function `getPadding()`
defined in ofLog.cpp as:

```c++
string & ofLog::getPadding() {
	static string padding = "";
	return padding;
}
```

**Comment by arturoc** [comment]
have you found any problem with this? the most correct in this cases is actually:

```cpp
string & ofLog::getPadding() {
    static string * padding = new string;
    return *padding;
}
```

to avoid it being accessed when already destroyed for example if you log from a destructor

**Comment by elliotwoods** [comment]
yes. when oF is statically linked to a dll which is dynamically linked by an application which also includes the oF headers
pretty esoteric use case i understand...
but this is one of the only issues so far

**Comment by arturoc** [comment]
:) can you send a PR? i suspect this will also fix #4482

**Comment by bakercp** [comment]
@elliotwoods Any update on this?  Have you been able to pull together a PR?
